app/helpers.php

```diff
@@ -117,12 +117,17 @@ function human_diff_string(\Carbon\Carbon $ts1, \Carbon\Carbon $ts2, $absolute =
         return 'unknown length';
     }
 
+    if ($ts2 <= \Carbon\Carbon::minValue() || $ts2 >= \Carbon\Carbon::maxValue()
```

Comment:
Values can be equal to the min/max, so less than/greater than should be used, rather than lt/gt or equal to.